src/Jobs/ProcessBatchTracking.php

```
@@ -22,274 +23,262 @@
+        $data['tenant_id'] = $data['tenant_id'] ?? $this->tenantId;
+        $data['site_id'] = $data['site_id'] ?? $this->siteId;
 
-		// Merge in common data
-		$data['ip_address'] = $data['ip_address'] ?? $this->ipAddress;
-		$data['user_agent'] = $data['user_agent'] ?? $this->userAgent;
-		$data['tenant_id']  = $data['tenant_id'] ?? $this->tenantId;
-		$data['site_id']    = $data['site_id'] ?? $this->siteId;
+        switch ($type) {
+            case 'pageview':
+                $pageViewData = $this->createPageViewData($data);
+                $provider->storePageView($pageViewData, $this->siteId);
```

PHPStan (2.2.7)

[error] Method ArtisanPackUI\Analytics\Providers\LocalAnalyticsProvider::storePageView() invoked with 2 parameters, 1 required.


(arguments.count)